modules/transactions/src/main/java/org/apache/ignite/internal/tx/storage/state/rocksdb/TxStateRocksDbStorage.java

```diff
@@ -297,19 +292,35 @@ public void close() throws Exception {
         }
     }
 
-    /** {@inheritDoc} */
     @Override
     public CompletableFuture<Void> flush() {
         return tableStorage.awaitFlush(true);
     }
 
-    /** {@inheritDoc} */
     @Override
     public long lastAppliedIndex() {
         return lastAppliedIndex;
     }
 
-    /** {@inheritDoc} */
+    @Override
+    public void lastAppliedIndex(long lastAppliedIndex) {
+        try (WriteBatch writeBatch = new WriteBatch()) {
```

Comment:
I don't really get why you need to have a batch here when you only write a single key-value pair

Reply:
Tried to fix it